src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
+
+  if (days < 3) {
+    return carRent * days;
+  } else if (days >= 3 && days < 7) {
```

Comment:
It's better to remove `else` because you use the `return` operator. Also remove `days >= 3`